skimage/filters/rank/generic.py

```diff
@@ -127,6 +129,30 @@ def _apply_vector_per_pixel(func, image, selem, out, mask, shift_x, shift_y,
     return out
 
 
+def default_selem(func):
```

Comment:
No, it's not the same, since we need a full structuring element for the median filter, while a star-shaped structuring element is used by default for mathematical morphology. I used the same name on purpose, for consistency. Is it a problem?